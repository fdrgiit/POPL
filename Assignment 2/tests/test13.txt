s #
a b c d e f #
s -> a b c #
s -> d e f #
##
