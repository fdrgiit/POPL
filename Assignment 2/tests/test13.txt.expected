FIRST(s) = { a, d }
FOLLOW(s) = { $ }
